.github/workflows/create-release-branch.yml

```diff
@@ -130,6 +133,15 @@ jobs:
                 }
               }
             }'
+      # Now that we've done our bookkeeping commits on the interim branch, use it as the base for the real release branch.
+      - name: Create release branch
+        id: create-branch
+        run: |
+          NEW_VERSION=$(jq -r ".version" package.json)
+          git checkout -b release-$NEW_VERSION
```

Comment:
Are we sure that this new release branch will be based off of the interim branch instead of dev?

Reply:
Since no other branch has been checked out in the meantime, it should be fine. But I'll do some testing in my side repo and confirm.

Reply:
Turns out there _was_ a problem here, but a different one.
Since the commit happens remotely, it's not actually recorded locally. So we do need to `git checkout -- .` and `git pull` in order to pull it in before we create the new branch.